hardware/raspberrypi/bootmodes/sdcard.md

```diff
@@ -1 +1,3 @@
-# SD Card boot
+# SD card boot
+
+The default way of using a Raspberry Pi is to boot it using an SD card: this is the recommended method for new and inexperienced users. See the [installation section of the documentation](https://www.raspberrypi.org/documentation/installation/) for more information on how to install a suitable operating system on an SD card.
```

Comment:
This should be a relative link, so that it works both on github and on the website